recipes/openscenegraph/all/conandata.yml

```diff
@@ -36,3 +36,13 @@ patches:
       patch_source:
         - https://github.com/openscenegraph/OpenSceneGraph/pull/1296
         - https://github.com/openscenegraph/OpenSceneGraph/pull/951
+    - patch_file: patches/fix-MSVC-_FPOSOFF-removal.patch
```

Comment:
The existing patches for `openscenegraph` are numbered (that is, they have a file name of the form `XXXX-YY.patch`, where `XXXX` is some number). However, I left the new patches in this PR unnumbered because of the fact that entries in the range [2, 4] are missing. Thus, there is some ambiguity regarding which numbers should be assigned to the new patches:

* `0012-fix-MSVC-_FPOSOFF-removal.patch` and `0013-fix-MSVC-unnamed-typedef-struct.patch` arise from starting after the last numbered patch, `0011-remove-deprecated-register.patch`.
* `0002-fix-MSVC-_FPOSOFF-removal.patch` and `0003-fix-MSVC-unnamed-typedef-struct.patch` arise from attempting to fill in the "gap" between `0001-fix-to_cmake_path-usage.patch` and `0005-use-JPEG-target-for-plugin.patch`.

What is the suggestion here?